docs/source/gbd2019_models/interventions/index.rst

```diff
@@ -1,16 +0,0 @@
-.. _2019_intervention_models:
```

Comment:
Why was this one deleted?

Reply:
No documents in these directories have ever been made

Reply:
🤷 okee this makes sense